src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -14,7 +14,8 @@ function init() {
     return;
   }
 
-  var mobileCarousel = new MobileCarousel();
+  var breakpointPx = 599;
```

Comment:
This isn't being reused, might as well just include it as a straight value in the argument below right?